website/docs/r/database_user.html.markdown

```diff
@@ -39,6 +39,11 @@ resource "mongodbatlas_database_user" "test" {
     key   = "My Key"
     value = "My Value"
   }
+
+  scopes {
+    name   = "My cluster name"
+    type = "CLUSTER"
+  }
```

Comment:
Can we include more than one scope in one of the examples?  I think that will be common and I know a few users were confused on how to do more than one label.

Reply:
sure, not problem, thank you